1.1/spec/11-Part-08.adoc

```diff
@@ -197,15 +258,30 @@ The `geo:hasSerialization` property is used to connect a geometry with its text-
 
 ```
 geo:hasSerialization a rdf:Property, 
-                       owl:DatatypePropterty;
-    rdfs:isDefinedBy <http://www.opengis.net/spec/geosparql/1.0>; 
+                       owl:DatatypeProperty;
+    rdfs:isDefinedBy <http://www.opengis.net/spec/geosparql/1.1>; 
     rdfs:label "has serialization"@en;
     rdfs:comment "Connects a geometry object with its text-based
                  serialization."@en;
     rdfs:domain geo:Geometry; 
     rdfs:range rdfs:Literal .
 ```
 
+===== Property: geo:inCRS
+
+The `geo:inCRS` property is used to connect a geometry with its text- based serialization (e.g., its WKT serialization).
```

Comment:
"The `geo:inCRS` property is used to connect a geometry with a text- based serialization of the CRS used for its representation (e.g. a WKT serialization)."

Reply:
Would this definition also include EPSG codes/EPSG URIs/SRID numbers? Or is there another property for this?

Reply:
Fixed by https://github.com/opengeospatial/ogc-geosparql/pull/72/commits/d0d40f5fcce89a516a7314b3048879f7ddf5814c

> Would this definition also include EPSG codes/EPSG URIs/SRID numbers? Or is there another property for this?

I've set the range value for the property to `skos:Concept`, so the object has to be a URI then, rather than a literal or even a microformat like a `dcterms:identifier` / `skos:notation`. I've asked the OGC already to update all their EPSG URIs like <http://www.opengis.net/def/crs/EPSG/0/4326> which currently only return XML, to returning Linked Data, so HTML & RDF as well as the current XML. When they do, those URIs will fit perfectly for use here. 

The OGC's data in that `/def/crs` resource is fine, it's just the presentation that's lacking...

We will surely too define some SHACL shapes for this 1.1 profile to apply further checks for appropriate values for things like this `inCRS` property.